### DISCUSSION TITLE
useAccount returns an undefined address

### DISCUSSION THREAD

**Discussion by DixonOrtiz** [wevm/wagmi#3028] [General]
Today I realized that the hook useAccount is not working as it did until yesterday. The first thing that I do is connect my Wallet with useWeb3Modal. Then I use the useAccount hook to extract the address but the result is always undefined, even when the login is done. 

Do you know anything that could help me to solve this?

![Captura de pantalla 2023-09-20 165122](https://github.com/wagmi-dev/wagmi/assets/36752730/dcbf0505-69d6-4d5f-9da7-146466b0e887)
![Captura de pantalla 2023-09-20 165127](https://github.com/wagmi-dev/wagmi/assets/36752730/e5f3b1e8-7eff-42a5-82cb-9cdff66e25a9)

**Comment by androidsixteen** [comment]
Did you figure this out @DixonOrtiz ?
